.github/workflows/VeniceCI-CompatibilityTests.yml

```diff
@@ -0,0 +1,264 @@
+# GitHub Actions workflow for running compatibility tests: Avro, Alpini unit, Alpini functional tests, and Pulsar Venice integration tests
+
+name:  TestsDeCompatibilite
```

Comment:
A little bit of French sprinkled in there, why not 😁

Reply:
haha. It helps to have tests in nice order under checks tab: unit --> e2e --> compatibility. Plus it sounds cool 😂